### DISCUSSION TITLE
What is the OCILIB equivalent to SYS_REFCURSOR of Pl/Sql

### DISCUSSION THREAD

**Discussion by aravindsreeram062** [vrogier/ocilib#319] [Q&A]
What is the OCILIB equivalent to SYS_REFCURSOR of Pl/Sql? if not, how do we access REFCURSOR of Pl/Sql in OCILIB?

**Comment by vrogier** [comment]
Hi,

OCILIB equivalent of SYS_REFCURSOR is OCI_Statement
Use OCI_GetStatement() for fetching OCI_BindStatement() for binding.
Check available [documentation ](http://vrogier.github.io/ocilib/doc/html/group___ocilib_c_api_fetching.html) (under section "Ref cursor Example") and [fetching code example](https://github.com/vrogier/ocilib/blob/master/demo/cursor.c) and [binding code example](https://github.com/vrogier/ocilib/blob/master/demo/ocilib_demo.c)

Regards,

Vincent

**Reply by aravindsreeram062** [reply]
Thank you Vincent, can we pass this OCI_GetStatement() as parameter to Pl/Sql procedure REFCURSOR?
